angularSTEP/src/app/recipe-page/recipe-page.component.spec.ts

```diff
@@ -1,25 +0,0 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Why are these two test files being deleted? Are their components already deleted? If not, please revert both

Reply:
I deleted them because they were giving errors that were unrelated to the component I was testing